### DISCUSSION TITLE
Using IconTintColorBehaviour on an ImageButton

### DISCUSSION THREAD

**Discussion by needledrag** [dotnet/maui#13640] [Q&A]
Does anyone know if there's a way to use the Community Toolkit IconTintColorBehaviour on an ImageButton?
Does a build run through and generate different icons for different tints to create the .png at build time?  

Or how to change the colour of an icon/image on an ImageButton?
Or is it only for an Image control, not ImageButton?

Regards

**Comment by kbsanders** [comment]
I'm also looking for an answer to this question, and not only for an `ImageButton` but a regular `Button` as well with an `ImageSource` property value specified.
